prow/config.yaml

```diff
@@ -4934,8 +4934,14 @@ presubmits:
       preset-service-account: true
     spec:
       containers:
-      - image: gcr.io/minigo-testing/minigo-prow-harness:latest
+      - image: gcr.io/minigo-testing/minigo-prow-harness-v2:latest
```

Comment:
can you link your Dockerfile?

Reply:
Not submitted yet: https://github.com/tensorflow/minigo/pull/79

Reply:
I guess... why not just use existing kubekins image? because of the pinned gcloud version?

Reply:
Oh, I dunno. I guess we could do that. But what about our other random deps that we need to install? Like Python 3.6

Reply:
you can build your image base off the existing images and add your deps?

Reply:
I think that's true, but that shouldn't affect this config/pr. We'll still need our own prow-harness-image and then we'll install our deps.

Reply:
Hahaha: 
https://github.com/kubernetes/test-infra/blob/a1a939885b73e8b180f49018ece25c3e3b9e94db/experiment/nursery/create_bazel_cache_rcs.sh#L51
You're not using bazel though right? :-)
Not a blocker, just amusing to immediately see someone violating this assumption.

Reply:
🤦‍♂️

Reply:
Ha!